### PR TITLE
AAQ-243: Refactor docs

### PR DESCRIPTION
**Pull request by sidravi1**
Reviewer: @suzinyou 

Estimate: 30 mins

----

# First pass at refactoring docs

## Ticket

Fixes: [AAQ-243](https://idinsight.atlassian.net/browse/AAQ-243)

## Description, Motivation and Context

Docs were getting a little messy and not very useful

### Goal

Improve layout, add relevant information.

### Changes

Reorganised content
Added new process flows and screenshots

### To do
1. Add configs relate to each of the components to the pages and describe what each does
2. Organise the "Other components" sections similar to "Core components"

## How Has This Been Tested?

Generated docs locally.

## Checklist:

This checklist is a useful reminder of small things that can easily be forgotten.
Put an `x` in all the items that apply and remove any items that are not relevant to this PR.

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [ ] I have updated the automated tests (if applicable)
- [x] I have written [good commit messages][1]
- [ ] I have updated the README file (if applicable)
- [x] I have updated affected documentation (if applicable)

[AAQ-243]: https://idinsight.atlassian.net/browse/AAQ-243?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/components/admin-app/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+The Admin App allows you to do the following:
+
+    https://[DOMAIN]/
+
+or if you are using the [dev](../../develop/setup.md) setup:
+
+    http://localhost:3000/
+
+## Features
+
```
Comment (suzinyou) [inline docs/components/admin-app/index.md:14]: Should we add access level here also? Otherwise we can only see this feature from left menu

Comment (suzinyou) [reply docs/components/admin-app/index.md:14]: Also, can we have cards here like you did for Core Components index? Seems more consistent and easy to view!

File: docs/components/qa-service/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+-   :material-api:{ .lg .middle .red} __Semantic Search__
+
+    ---
+
+    Allows searching through the content database. Use similarity between
+    embeddings of questions and the content to return the best matches
+
+    [:octicons-arrow-right-24: More info](./semantic-search.md)
+
+-   :material-react:{ .lg .middle } __LLM Response__
```
Comment (suzinyou) [inline docs/components/qa-service/index.md:17]:
```suggestion
-   :material-api:{ .lg .middle } __LLM Response__
```

File: mkdocs.yml [hunk 3]
```diff
@@ -30,38 +33,56 @@ theme:
...
+          - Access Levels: components/admin-app/access-level.md
+          - Manage Content: components/admin-app/manage-content.md
+          - Playground: components/admin-app/playground.md
+      - Question-Answering Service:
+          - components/qa-service/index.md
+          - Semantic Search: components/qa-service/semantic-search.md
+          - LLM Response: components/qa-service/llm-response.md
+      - Triaging Service:
+          - components/triaging/index.md
+  - Other Components:
```
Comment (suzinyou) [inline mkdocs.yml:52]: Would it be easy to set up an index.md for Other components too?
Right now when I click this menu, I am directed to WhatsApp/setup.md which can be confusing!

I really like the card style display of features/components, so in this index.md we can have Whatsapp, Chat manager, Offline LLM cards?

**Review by suzinyou — CHANGES_REQUESTED** [review]
Thank you Sid!!! 🎆

File: docs/components/qa-service/swagger-ui-screenshot.png
[the remark this answers is not in the document]
Comment (sidravi1) [reply docs/components/qa-service/swagger-ui-screenshot.png:1]: Oh yeah. This was just a visual to break up all the text. But it is a bit hard to read. Clicking on it makes it bigger but not sure if that helps on your mobile.

I can make it a bit skinner.

File: docs/components/admin-app/index.md (hunk 1 above)
Comment (sidravi1) [reply docs/components/admin-app/index.md:14]: I can do cards. I was just being lazy 😛 

Re: access level - it's not really a capability. But I will mention it on this page so there is another reference to it

File: mkdocs.yml (hunk 3 above)
Comment (sidravi1) [reply mkdocs.yml:52]: Done!

**Review by suzinyou — APPROVED** [review]
Amazing thank you!

**Pull request merged**